week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,8 +9,17 @@
 */
 
 function calculateGrade(mark) {
+    if (mark >= 80) {
+      return "A";
+    } else if(mark>60) {
+      return "B";
+    } else if (mark>50){
```

Comment:
Good! Can you tell me if these conditions have to appear in this particular sequence? What happens when we move them around?